File: app/src/main/java/com/tistory/black_jin0427/myandroidarchitecture/api/GithubApi.java
package com.tistory.black_jin0427.myandroidarchitecture.api;

import com.tistory.black_jin0427.myandroidarchitecture.api.model.UserResponse;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Url;

public interface GithubApi {

    @GET
    Observable<UserResponse> getUserList(@Url String url);
}

File: app/src/main/java/com/tistory/black_jin0427/myandroidarchitecture/api/GithubApi.kt
package com.tistory.black_jin0427.myandroidarchitecture.api

import com.tistory.black_jin0427.myandroidarchitecture.api.model.UserResponse
import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Url

interface GithubApi {

    @GET
    fun getUserList(@Url url: String): Observable<UserResponse>
}
